Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'random_data'
-gem 'bootstrap-sass'
+# gem 'bootstrap-sass'
```

Comment:
It's odd to just comment out Gem's.  If things are commented out and not deleted, generally it's best to leave a note as to why.  Leaves future developers who aren't you with the knowledge of WHY.